Documentation for compression feature

CourtneyThurston opened the pull request:
Detailing the compression feature rolling out soon. Framework changes have already been deployed; service layer and portal are about to be deployed. 

I'm the intern who developed this feature; feel free to ping t-cothu with questions (here until the 18th).

azurecla commented:
Hi __@gperrot__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

CourtneyThurston committed:
Documentation for compression feature

Detailing the compression feature rolling out soon. Framework changes have already been deployed; service layer and portal are in deployment stages.

minhe-msft commented on articles/stream-analytics/stream-analytics-define-inputs.md:
```diff
@@ -53,6 +59,7 @@ The following table explains each property in the **New input** blade in the Azu
 | **Event hub consumer group** (optional) |The consumer group to use to ingest data from the event hub. If no consumer group is specified, the Stream Analytics job uses the default consumer group. We recommend that you use a distinct consumer group for each Stream Analytics job. |
 | **Event serialization format** |The serialization format (JSON, CSV, or Avro) of the incoming data stream. |
 | **Encoding** | UTF-8 is currently the only supported encoding format. |
+| **Compression** (optional) | The compression type (None, GZip, or Deflate) of the incoming data stream. |
```
Since this compression setting is optional, please call out what is the default value.

minhe-msft commented on articles/stream-analytics/stream-analytics-define-inputs.md:
```diff
@@ -98,6 +105,7 @@ The following table explains each property in the **New input** blade in the Azu
 | **Consumer group** (optional) |The consumer group to use to ingest data from the IoT hub. If no consumer group is specified, a Stream Analytics job uses the default consumer group. We recommend that you use a different consumer group for each Stream Analytics job. |
 | **Event serialization format** |The serialization format (JSON, CSV, or Avro) of the incoming data stream. |
 | **Encoding** |UTF-8 is currently the only supported encoding format. |
+| **Compression** (optional) | The compression type (None, GZip, or Deflate) of the incoming data stream. |
```
Since this compression setting is optional, please call out what is the default value.

minhe-msft commented on articles/stream-analytics/stream-analytics-define-inputs.md:
```diff
@@ -140,6 +148,7 @@ The following table explains each property in the **New input** blade in the Azu
 | **Time format** (optional) |  If you use the time variable in the path, the time format in which the files are organized. Currently the only supported value is `HH`. |
 | **Event serialization format** | The serialization format (JSON, CSV, or Avro) for incoming data streams. |
 | **Encoding** | For CSV and JSON, UTF-8 is currently the only supported encoding format. |
+| **Compression** (optional) | The compression type (None, GZip, or Deflate) of the incoming data stream. |
```
Since this compression setting is optional, please call out what is the default value.
